### PR TITLE
ALBS-165: Added logic for detecting not updated module streams

### PR DESCRIPTION
**Pull request by anfimovdm**

### CODE REVIEW

File: alws/build_planner.py [hunk 1]
```diff
@@ -159,29 +163,64 @@ async def add_task(self, task: build_schema.BuildTaskRef):
...
+                    f'/api/v1/distros/{clean_dist_name}/{distr_ver}'
+                    f'/module/{module.name}/{module.stream}/{arch}/'
+                )
+                beholder_response = None
+                pkgs_to_add = []
+                if not self._skip_module_checking:
+                    try:
+                        beholder_response = await beholder_client.get(endpoint)
+                    except Exception:
+                        logging.error('Cannot get module info')
```
Comment (AvoidMe) [inline alws/build_planner.py:201]: You can add `continue` here and remove `if beholder_response is not None:`

File: alws/build_planner.py [hunk 2]
```diff
@@ -159,29 +163,64 @@ async def add_task(self, task: build_schema.BuildTaskRef):
...
+                        for item in beholder_response.get('components', [])
+                    }
+                    for ref_id, component_name in beholder_components.items():
+                        git_ref_id = None
+                        try:
+                            git_ref_id = await self.get_ref_commit_id(
+                                component_name, module.stream)
+                        except Exception:
+                            logging.exception('Cannot get git_ref_commit_id:')
+                        if git_ref_id is not None and git_ref_id == ref_id:
```
Comment (AvoidMe) [inline alws/build_planner.py:214]: Same here

File: alws/build_planner.py (hunk 1 above)
Comment (anfimovdm) [reply alws/build_planner.py:200]: Fixed

File: alws/build_planner.py (hunk 2 above)
Comment (anfimovdm) [reply alws/build_planner.py]: Fixed

**Review by AvoidMe — APPROVED** [review]

**Pull request merged**